testsuite/tests/ui/applications/test_application_plans.py

```diff
@@ -0,0 +1,51 @@
+"""Tests of applications plans"""
+import pytest
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+
+from testsuite import rawobj
+from testsuite.utils import blame
+from testsuite.ui.views.admin.product.application import ApplicationPlansView
+from testsuite import TESTED_VERSION  # noqa # pylint: disable=unused-import
+
+
+def test_app_plan_create(custom_ui_app_plan, request, service):
+    """
+    Test:
+        - create application plan via UI
+        - assert that plan was created
+    """
+    name = blame(request, "app-plan")
+
+    plan = custom_ui_app_plan(name, service)
+    assert plan
+    assert plan['name'] == name
+
+
+# pylint: disable=unused-argument
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-825")
+@pytest.mark.skipif("TESTED_VERSION < Version('2.13')")
+def test_default_app_plan_change(login, service, custom_app_plan, navigator):
+    """
+    Test cover functionality of  feature 825 - default application plan change
```

Comment:
We don't need to mention the issue here. that is what the mark issue is for.
I'd rather see short description.

Reply:
Yes, we need it. It is a feature request and needs to be tracked.